ui/src/components/Equipment/PlateManipulator.jsx

```diff
@@ -634,33 +634,23 @@ export default function PlateManipulator(props) {
     >
       <Col className="ms-3">
         <ButtonToolbar className="ms-4">
-          {!inPopover ? (
+          {!inPopover && (
             <div className="me-4">
               <b>{cplate_label}</b>
             </div>
-          ) : null}
-          <OverlayTrigger
-            variant="outline-success"
-            placement="bottom"
-            overlay={
-              <Tooltip id="select-samples">
-                Refresh if Plate Location not Updated
-              </Tooltip>
-            }
+          )}
+          <TooltipTrigger
+            id="refresh-tooltip"
```

Comment:
I changed all the IDs as well, since some of them had been copy-pasted and did not match the purpose of the tooltip.

Reply:
When we make the switch to React 18, we can generate random IDs with [`useId()`](https://react.dev/reference/react/useId)